plugin/evm/atomic/state/atomic_trie_test.go

```diff
@@ -904,9 +819,76 @@ func benchmarkApplyToSharedMemory(b *testing.B, disk database.Database, blocks u
 	b.ReportAllocs()
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-		backend.(*atomicBackend).sharedMemory = testSharedMemory()
+		backend.sharedMemory = atomictest.TestSharedMemory()
 		assert.NoError(b, backend.MarkApplyToSharedMemoryCursor(0))
 		assert.NoError(b, db.Commit())
 		assert.NoError(b, backend.ApplyToSharedMemory(lastAcceptedHeight))
 	}
 }
+
+// verifyOperations creates an iterator over the atomicTrie at [rootHash] and verifies that the all of the operations in the trie in the interval [from, to] are identical to
+// the atomic operations contained in [operationsMap] on the same interval.
+func verifyOperations(t testing.TB, atomicTrie *AtomicTrie, codec codec.Manager, rootHash common.Hash, from, to uint64, operationsMap map[uint64]map[ids.ID]*avalancheatomic.Requests) {
```

Comment:
A few nits for this function, since it shows as new code from a git perspective. I understand it's meant to just be moved though, so feel free to ignore the few nits.

First nit, should we rename `operationsMap` to `heightToBlockRequests` (given the var names in the for loop, and also no point stating it's a map I think)